index.html

```diff
@@ -29,7 +29,46 @@ <h1 class="text-center landing">Introducing Gaffer</h1>
 </section>
 <!-- End About Section-->
 
-<hr />
+<!-- Tour Section-->
+<section id="tour" class="container ptb-60 wow">
+    <div class="row ptb-15 wow">
+        <div class="col-xs-12 col-md-6">
+            <!-- Lost in Space S1 forest robot grey -->
+            <a class="cbox-gallary1" href="http://image-engine.com/wp-content/uploads/2018/04/101_040_600_AO.1006.jpg">
+                <div class="bg-img" style="height: 300px; background-image: url(http://image-engine.com/wp-content/uploads/2018/04/101_040_600_AO.1006.jpg);"></div>
+            </a>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <h3 class="mt-60 text-center">Create Scenes</h3>
+            <p>Import, build, and manipulate scenes non-destructively. Duplicate, combine, join, prune, split, and inspect scenes of arbitrary complexity. Iterate over your sequence's components in a fast and frustration-free interface.</p>
+        </div>
+    </div>
+    <div class="row ptb-15 wow">
+        <div class="col-xs-12 col-md-6 pull-right">
+            <!-- Shader graph -->
+            <a class="cbox-gallary1" href="img/headers/header_graph.jpg">
+                <div class="bg-img" style="height: 300px; background-image: url(img/headers/header_graph.jpg); background-position: right"></div>
+            </a>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <h3 class="mt-60 text-center">Build Shaders</h3>
+            <p>With full support for <a href="https://www.solidangle.com/arnold" alt="Arnold homepage">Arnold</a> and <a href="http://www.3delight.com" alt="3Delight homepage">3Delight's</a> powerful industry-standard shaders, use the robust node plug-outs to combine and design. For extra customization, arrange and write custom shaders using <a href="http://opensource.imageworks.com/?p=osl" alt="Open Shading Language homepage">Open Shading Language</a> nodes and code.</p>
```

Comment:
"use the robust node plug-outs to combine and design." I don't understand this statement